packages/dream-html/dream-html.0.0.2/opam

```diff
@@ -0,0 +1,40 @@
+opam-version: "2.0"
+synopsis: "HTML generator eDSL for Dream"
+description:
+  "Write HTML directly in your OCaml source files with editor support."
+maintainer: ["Yawar Amin <yawar.amin@gmail.com>"]
+authors: ["Yawar Amin <yawar.amin@gmail.com>"]
+license: "GPL-3.0-or-later"
+homepage: "https://github.com/yawaramin/dream-html"
+doc: "https://yawaramin.github.io/dream-html/"
+bug-reports: "https://github.com/yawaramin/dream-html/issues"
+depends: [
+  "dune" {>= "2.7"}
+  "ocaml" {>= "4.08.0"}
+  "dream"
```

Comment:
```suggestion
  "dream" {"1.0.0~alpha3"}
```
For
```
#=== ERROR while compiling dream-html.0.0.2 ===================================#
# context              2.2.0~alpha~dev | linux/x86_64 | ocaml-base-compiler.4.08.1 | pinned(https://github.com/yawaramin/dream-html/releases/download/v0.0.2/dream-html-0.0.2.tbz)
# path                 ~/.opam/4.08/.opam-switch/build/dream-html.0.0.2
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p dream-html -j 47 @install
# exit-code            1
# env-file             ~/.opam/log/dream-html-7-e46207.env
# output-file          ~/.opam/log/dream-html-7-e46207.out
### output ###
# File "/home/opam/.opam/4.08/lib/biniou/biniou.dune", line 1, characters 0-0:
# Warning: .dune files are ignored since 2.0. Reinstall the library with dune
# >= 2.0 to get rid of this warning and enable support for the subsystem this
# library provides.
# File "/home/opam/.opam/4.08/lib/faraday/faraday.dune", line 1, characters 0-0:
# Warning: .dune files are ignored since 2.0. Reinstall the library with dune
# >= 2.0 to get rid of this warning and enable support for the subsystem this
# library provides.
# File "/home/opam/.opam/4.08/lib/faraday-lwt-unix/faraday-lwt-unix.dune", line 1, characters 0-0:
# Warning: .dune files are ignored since 2.0. Reinstall the library with dune
# >= 2.0 to get rid of this warning and enable support for the subsystem this
# library provides.
# File "/home/opam/.opam/4.08/lib/faraday-lwt/faraday-lwt.dune", line 1, characters 0-0:
# Warning: .dune files are ignored since 2.0. Reinstall the library with dune
# >= 2.0 to get rid of this warning and enable support for the subsystem this
# library provides.
# File "/home/opam/.opam/4.08/lib/lwt_ssl/lwt_ssl.dune", line 1, characters 0-0:
# Warning: .dune files are ignored since 2.0. Reinstall the library with dune
# >= 2.0 to get rid of this warning and enable support for the subsystem this
# library provides.
#       ocamlc lib/.dream_html.objs/byte/dream_html.{cmi,cmti} (exit 2)
# (cd _build/default && /home/opam/.opam/4.08/bin/ocamlc.opt -w -40 -g -bin-annot -I lib/.dream_html.objs/byte -I /home/opam/.opam/4.08/lib/angstrom -I /home/opam/.opam/4.08/lib/base64 -I /home/opam/.opam/4.08/lib/bigarray-compat -I /home/opam/.opam/4.08/lib/bigstringaf -I /home/opam/.opam/4.08/lib/biniou -I /home/opam/.opam/4.08/lib/bytes -I /home/opam/.opam/4.08/lib/caqti -I /home/opam/.opam/4.08/lib/caqti-lwt -I /home/opam/.opam/4.08/lib/cstruct -I /home/opam/.opam/4.08/lib/digestif -I /home/opam/.opam/4.08/lib/dream -I /home/opam/.opam/4.08/lib/dream/cipher -I /home/opam/.opam/4.08/lib/dream/graphiql -I /home/opam/.opam/4.08/lib/dream/graphql -I /home/opam/.opam/4.08/lib/dream/http -I /home/opam/.opam/4.08/lib/dream/localhost -I /home/opam/.opam/4.08/lib/dream/middleware -I /home/opam/.opam/4.08/lib/dream/pure -I /home/opam/.opam/4.08/lib/dream/sql -I /home/opam/.opam/4.08/lib/duration -I /home/opam/.opam/4.08/lib/easy-format -I /home/opam/.opam/4.08/lib/eqaf -I /home/opam/.opam/4.08/lib/eqaf/bigstring -I /home/opam/.opam/4.08/lib/eqaf/cstruct -I /home/opam/.opam/4.08/lib/faraday -I /home/opam/.opam/4.08/lib/faraday-lwt -I /home/opam/.opam/4.08/lib/faraday-lwt-unix -I /home/opam/.opam/4.08/lib/fmt -I /home/opam/.opam/4.08/lib/gluten -I /home/opam/.opam/4.08/lib/gluten-lwt -I /home/opam/.opam/4.08/lib/gluten-lwt-unix -I /home/opam/.opam/4.08/lib/graphql -I /home/opam/.opam/4.08/lib/graphql-lwt -I /home/opam/.opam/4.08/lib/graphql_parser -I /home/opam/.opam/4.08/lib/h2 -I /home/opam/.opam/4.08/lib/h2-lwt -I /home/opam/.opam/4.08/lib/h2-lwt-unix -I /home/opam/.opam/4.08/lib/hmap -I /home/opam/.opam/4.08/lib/hpack -I /home/opam/.opam/4.08/lib/httpaf -I /home/opam/.opam/4.08/lib/httpaf-lwt -I /home/opam/.opam/4.08/lib/httpaf-lwt-unix -I /home/opam/.opam/4.08/lib/logs -I /home/opam/.opam/4.08/lib/lwt -I /home/opam/.opam/4.08/lib/lwt/unix -I /home/opam/.opam/4.08/lib/lwt_ssl -I /home/opam/.opam/4.08/lib/magic-mime -I /home/opam/.opam/4.08/lib/mirage-crypto -I /home/opam/.opam/4.08/lib/mirage-crypto-rng -I /home/opam/.opam/4.08/lib/mirage-crypto-rng/lwt -I /home/opam/.opam/4.08/lib/mirage-crypto-rng/unix -I /home/opam/.opam/4.08/lib/mmap -I /home/opam/.opam/4.08/lib/mtime -I /home/opam/.opam/4.08/lib/mtime/os -I /home/opam/.opam/4.08/lib/multipart-form-data -I /home/opam/.opam/4.08/lib/ocaml/threads -I /home/opam/.opam/4.08/lib/psq -I /home/opam/.opam/4.08/lib/ptime -I /home/opam/.opam/4.08/lib/re -I /home/opam/.opam/4.08/lib/result -I /home/opam/.opam/4.08/lib/rresult -I /home/opam/.opam/4.08/lib/seq -I /home/opam/.opam/4.08/lib/ssl -I /home/opam/.opam/4.08/lib/stdlib-shims -I /home/opam/.opam/4.08/lib/stringext -I /home/opam/.opam/4.08/lib/uri -I /home/opam/.opam/4.08/lib/websocketaf -I /home/opam/.opam/4.08/lib/yojson -no-alias-deps -o lib/.dream_html.objs/byte/dream_html.cmi -c -intf lib/dream_html.mli)
# File "lib/dream_html.mli", line 63, characters 12-24:
# 63 | val write : Dream.stream -> node -> unit Dream.promise
#                  ^^^^^^^^^^^^
# Error: Unbound type constructor Dream.stream
```